config/config.go

```diff
@@ -32,7 +32,7 @@ var Filename string
 
 // ReadConfig reads in config file and ENV variables if set.
 func ReadConfig() {
-	filename, found := os.LookupEnv("CIVOCONFIG")
+	filename, found := os.LookupEnv("CIVO_CONFIG")
```

Comment:
I do think this is better but little worried that this could be a breaking change if someone already relied on it.

Reply:
@RealHarshThakur  thats fair point but I am wondering if anyones is using it as until someone digs the code the will not know if such an env variable exists. Thats why I added another issue to update the README with usage. WDYT?

Reply:
Yep, agreed it's unlikely since it wasn't documented.